rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1268,6 +1268,19 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 			m_gui_settings->SetValue(gui::gs_showMouseFs, val);
 		});
 
+		ui->gs_hideMouseOnIdle->setChecked(m_gui_settings->GetValue(gui::gs_hideMouseIdle).toBool());
+		connect(ui->gs_hideMouseOnIdle, &QCheckBox::clicked, [this](bool val)
```

Comment:
the bool is called "checked".
You may correct this on other places

Reply:
Do you mean the "&QCheckBox::clicked" part?
I can't see an equivalent for 'checked', there are some instances using "&QCheckBox::stateChanged", would you prefer that?
I think probably better if I just updated this usage here, and did another PR to convert the others if desired.

Reply:
we don't accept PRs like that

Reply:
clicked is a protected function `QAbstactButton::clicked(bool checked)`